README.md

```diff
@@ -68,18 +68,21 @@ Create a json configuration file in your project:
 ```json
 {
   "parsing": {
-    "source": {
-      "default": ["path/to/interfaces.ts"]
+    "targets": {
+      "default": {
+        "source": ["path/to/interfaces.ts"]
+      }
     }
   },
   "rendering": {
     "swift": {
-      "templates": {
-        "default": "path/to/module-interface.mustache"
-      },
-      "outputPath": {
-        "default": "path/to/output/directory"
-      },
+      "renders": [
```

Comment:
Maybe we could merge parsing and rendering, then it could be 
```
targets: { 
  default:  {
    source:
    templates: 
    outputPath:
  }
}
```

Reply:
`rendering` supports multiple languages, and also it can render multiple different results based on a single parsed target. That's why it is converted to an array.